src/main/java/mate/academy/hibernate/relations/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/orm/cfg">
```

Comment:
this is not java code
![image](https://user-images.githubusercontent.com/20326858/221166913-80f4e333-edb9-4b4f-b846-400d732eaa0f.png)
![image](https://user-images.githubusercontent.com/20326858/221167088-f4b76b3b-e733-46e9-8771-048c89d9059e.png)